runtime/oti/jvminit.h

```diff
@@ -391,6 +391,9 @@ enum INIT_STAGE {
 #define VMOPT_XXDISABLEVALHALLA "-XX:-EnableValhalla"
 #endif /* defined(J9VM_OPT_VALHALLA_VALUE_TYPES) */
 
+#define VMOPT_XXADDVALUEBASEDCLASSCHECKSEXCEPTION "-XX:addValueBasedClassChecks=exception"
+#define VMOPT_XXADDVALUEBASEDCLASSCHECKSWARN "-XX:addValueBasedClassChecks=warn"
```

Comment:
Are we matching the Hotspot option here?   It would be best if we can use the same option so users get a consistent experience.

 If not, our `-XX` options start with capital letters (mostly).  Renaming to something like `-XX:ValueBasedClassCheck=[exception|warn]` would be clearer?  

Or handled with multiple options like:
`-XX[+-]EnableValueBasedClassChecks` to enable the checks
`-XX:[+-]ThrowForValueBasedClassChecks` to throw an exception
`-XX:[+-]WarnForValueBasedClassChecks` to warn

Reply:
> Are we matching the Hotspot option here? It would be best if we can use the same option so users get a consistent experience.

I agree to use the same CML as RI. Seems that RI has not finished implementing this JEP yet. We can change this CML to match RI once we know what they use.

Reply:
Updated the CML to `-XX:ValueBasedClassCheck=[exception|warn]`